servers/rendering/renderer_rd/shaders/effects/tonemap.glsl

```diff
@@ -498,5 +492,11 @@ void main() {
 		color.rgb = apply_color_correction(color.rgb);
 	}
 
+	if (params.use_debanding) {
```

Comment:
Doing this means debanding is applied *after* tonemapping – can this be a problem with certain tonemappers?

Reply:
No, it shouldn't be a problem. Debanding should not be done before tonemapping anyway regardless of the tonemapper used. Debanding needs to be done right before quantizing.